content/applications/general/users.rst

```diff
@@ -1,161 +1,20 @@
-:show-content:
+:nosearch:
+:hide-page-toc:
+:show-toc:
```

Comment:
@AntoineVDV 

Since this page isn't displayed at all, I remove the paragraph below and this `show-toc` tag.